Gemfile.lock

```diff
@@ -9,7 +9,8 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    aasm (4.11.1)
+    aasm (4.12.3)
+      concurrent-ruby (~> 1.0)
```

Comment:
Where are these changes coming from?

Reply:
I was getting weird dependency errors because of changes in this file.
This is a little suspect because there's no equivalent changes to the `Gemfile`.  Since we're already handling this in #386, I'd like for these changes to be reverted.